src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,65 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.*;
+/**
+ * Represents block number of a person's address
+ * @author langyanbin0314
+ *
+ */
+
+public class Block {
+	
+	public static final String EXAMPLE = "17";
+	public static final String MESSAGE_BLOCK_CONSTRAINT = "Block number should be a positive integer";
+	public static final String BLOCK_VALIDATION_REGEX = "\\d+";
+	
+	public final String value;
+	private boolean isPrivate;
+	
+	/**
+	 * Checks whether the given input is a valid block number
+	 * @param input
+	 * @return
```

Comment:
Normally, you should give a description of the `@param` or the `@return` value. If you have nothing more to add, probably the comment line can be deleted as well (because it's not adding any value).